cindy/index.js

```diff
@@ -0,0 +1,8 @@
+// *** Task // with Math.random() ==> get random item from colors array, if the item is number set it to ''
+
+const colors = ["red", 9999, "green", "blue",777, "yellow", "black", "purple", "brown"];
+
+
+const colors2 = colors[Math.floor(colors.length * Math.random())];
+colors3 = `${colors2}`
```

Comment:
You still get numbers in console! After line 6 you have to filter numbers so you have to use if statement and the logic is, if you faced number from randomized variable which is colors3 you have to change it to an empty string! Try to find solution and if you had no clues just write me and we make a BO!